dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -561,38 +561,48 @@ dlfUtils.isCorsEnabled = function(imageObjs) {
     // fetch data from server
     // with access control allowed
     let response = true;
-
+    const type = 'HEAD';
     imageObjs.forEach(function(imageObj) {
         let url = imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.ZOOMIFY
-            ? imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg')
-            :
-            imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
-                ? dlfViewerSource.IIIF.getMetdadataURL(imageObj.url)
+            ? dlfUtils.buildXhrURI(imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg'), true)
+            : imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
+                ? dlfUtils.buildXhrURI(dlfViewerSource.IIIF.getMetdadataURL(imageObj.url), false)
```

Comment:
The problem with `mixed_content` ist, hat running TYPO3 on HTTPS fetching images via HTTP the isCorsTest always fails as the ajax-call is rejected. But that's not the intention of this function.
Fetching the HEAD is IMHO only possible via server as we have done it before your patch.

We only need this isCorsEnabled to be sure, the image manipulation on WebGL will work. But this even works with `mixed_contents` if the CORS-headers are set.

Your current idea does not work for static images as you fetch the HEAD of the server via eID-script not of the image.

Maybe you get my point by an example:

Call it via HTTP or HTTPS - the images are provided with CORS header via HTTP:

http://test.dfg-viewer.de/show/?tx_dlf%5Bid%5D=http%3A%2F%2Fdigital.slub-dresden.de%2Ffileadmin%2Fdata%2F351357262%2Faufdesun_351357262_mets.xml&no_cache=1

That's what isCorsEnabled fetches and always returns false:

<pre>
Request URL:https://test.dfg-viewer.de/show/?eID=tx_dlf_geturl_eid&url=http%3A%2F%2Fdigital.slub-dresden.de%2Ffileadmin%2Fdata%2F351357262%2F351357262_tif%2Fjpegs%2F00000001.tif.large.jpg&header=2
Request Method:HEAD
Status Code:200 OK
Referrer Policy:no-referrer-when-downgrade
</pre>